BUILD.md

```diff
@@ -295,3 +298,59 @@ Module names can be found [here](bin/run_lint#L205) in the codebase.
   then you need to set `LD_LIBRARY_PATH` with:
   `source /opt/intel/sgxsdk/environment` .
   For details, see [PREREQUISITES](PREREQUISITES.md)
+
+# <a name="avaloncentos8"></a>Run Avalon on CentOS 8.2
+## <a name="sgxinstall"></a>SGX driver and PSW package installation
+1. Install SGX drivers
+   ```
+   cd /tmp
+   sudo yum update
+   sudo yum install kernel-devel gcc
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_linux_x64_driver_2.6.0_602374c.bin
+   chmod +x sgx_linux_x64_driver_2.6.0_602374c.bin
+   sudo ./sgx_linux_x64_driver_2.6.0_602374c.bin
+   
+   You should be able to see /dev/isgx device mapping on your host.
+   ```
+2. Install SGX PSW package
+   ```
+   cd /tmp
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_rpm_local_repo.tgz
+   tar -xvf sgx_rpm_local_repo.tgz
+   yum-config-manager --add-repo file:///tmp/sgx_rpm_local_repo
+   cd sgx_rpm_local_repo/
+   sudo yum update -y
+   sudo yum --nogpgcheck install -y libsgx-launch libsgx-epid libsgx-quote-ex libsgx-urts
+   
+   You can check the aesm service should be up and running:
+     sudo systemctl status aesmd.service
+
+## <a name="runavalon"></a>Run Avalon
+This section covers to run Avalon on CentOS 8.2 host in docker mode.
+Prepare a host with CentOS 8.2 and install docker, docker-compose on it. Clone the hyperledger/avalon repository and go to
+avalon directory. Change the parameter DISTRO to 'centos' and IMAGE to centos:centos8 in .env file.
+
+To build and run in direct model (SGX-SIM), execute the following command -
+```
+docker-compose -f docker-compose.yaml up --build
+```
+
+To build and run in direct model (SGX-HW) , execute the following command -
+```
+docker-compose -f docker-compose.yaml -f docker-compose-sgx.yaml up --build
+```
+
+To build and run in fabric proxy  model, execute the following command -
```

Comment:
It would be worthy to mention the proxy setup as well before running the compose setup.